chromadb/api/__init__.py

```diff
@@ -378,3 +378,23 @@ def get_settings(self) -> Settings:
 
         """
         pass
+
+    @abstractmethod
+    def get_system_info(
```

Comment:
can we add chroma version, and chroma config?

Reply:
Done.

```json
{
  "chroma_version": "0.4.10",
  "chroma_settings": {
    "environment": "",
    "chroma_db_impl": null,
    "chroma_api_impl": "chromadb.api.segment.SegmentAPI",
    "chroma_telemetry_impl": "chromadb.telemetry.posthog.Posthog",
    "chroma_sysdb_impl": "chromadb.db.impl.sqlite.SqliteDB",
    "chroma_producer_impl": "chromadb.db.impl.sqlite.SqliteDB",
    "chroma_consumer_impl": "chromadb.db.impl.sqlite.SqliteDB",
    "chroma_segment_manager_impl": "chromadb.segment.impl.manager.local.LocalSegmentManager",
    "tenant_id": "default",
    "topic_namespace": "default",
    "is_persistent": true,
    "persist_directory": "/chroma/chroma",
    "chroma_server_host": null,
    "chroma_server_headers": null,
    "chroma_server_http_port": null,
    "chroma_server_ssl_enabled": false,
    "chroma_server_api_default_path": "/api/v1",
    "chroma_server_grpc_port": null,
    "chroma_server_cors_allow_origins": [],
    "chroma_server_auth_provider": "",
    "chroma_server_auth_configuration_provider": null,
    "chroma_server_auth_configuration_file": null,
    "chroma_server_auth_credentials_provider": "*****",
    "chroma_server_auth_credentials_file": "*****",
    "chroma_server_auth_credentials": "*****",
    "chroma_client_auth_provider": null,
    "chroma_server_auth_ignore_paths": {
      "/api/v1": [
        "GET"
      ],
      "/api/v1/heartbeat": [
        "GET"
      ],
      "/api/v1/version": [
        "GET"
      ]
    },
    "chroma_client_auth_credentials_provider": "*****",
    "chroma_client_auth_protocol_adapter": "chromadb.auth.providers.RequestsClientAuthProtocolAdapter",
    "chroma_client_auth_credentials_file": "*****",
    "chroma_client_auth_credentials": "*****",
    "chroma_client_auth_token_transport_header": null,
    "chroma_server_auth_token_transport_header": null,
    "chroma_server_env_endpoint_enabled": true,
    "anonymized_telemetry": true,
    "allow_reset": false,
    "migrations": "apply"
  },
  "datetime": "2023-09-16T10:58:32.271390",
  "persist_directory": "/chroma/chroma",
  "python_version": "3.10.13",
...
}
```